pokecli.py

```diff
@@ -341,6 +341,8 @@ def init_config():
     config.hatch_eggs = load.get("hatch_eggs", True)
     config.longer_eggs_first = load.get("longer_eggs_first", True)
 
+    config.forts_show_name = load.get('forts', {}).get('show_name', False)
```

Comment:
Please use the convention of calling `add_config` to set up this. Check how other parameters that are inside the `forts` group are used.